logstash-core/spec/logstash/modules/scaffold_spec.rb

```diff
@@ -222,9 +222,9 @@
     let(:base_dir) { File.expand_path(File.join(File.dirname(__FILE__), "..", "..", "modules_test_files", "#{mname}")) }
     let(:module_settings) do
       {
-        "var.output.elasticsearch.hosts" => "localhost:9200",
-        "var.output.elasticsearch.user" => "foo",
-        "var.output.elasticsearch.password" => "password",
+        "var.elasticsearch.hosts" => "localhost:9200",
+        "var.elasticsearch.user" => "foo",
+        "var.elasticsearch.password" => "password",
         "var.input.tcp.port" => 5606,
         "dashboards.kibana_index" => ".kibana"
```

Comment:
Only if you make other changes - this line can be removed now, sorry I overlooked it.